external/synth/synth/schemas.py

```diff
@@ -10,3 +10,9 @@ def load_schema(name: str) -> DatasetSchema:
     path = os.path.join(CURRENT_DIR, "_dataset_fixtures", name)
     with open(path) as f:
         return load(f)
+
+
+def load_schema_directory(name: str) -> DatasetSchema:
```

Comment:
It feels confusing to have one function named `load_schema_directory` call another called `load_directory_schema`.  Maybe a more descriptive name could work better?
```suggestion
def load_directory_schema_from_library(name: str) -> DatasetSchema:
```